src/ui/components/results/universalresultscomponent.js

```diff
@@ -67,7 +67,7 @@ export default class UniversalResultsComponent extends Component {
       // Show a view more link by default, which also links to verticalURL.
       viewMore: true,
       // By default, the view more link has a label of 'View More'.
-      viewMoreLabel: config.viewAllText || 'View More',
+      viewMoreLabel: defaultConfigOption(config, ['viewMoreLabel', 'viewAllText'], 'View More'),
```

Comment:
i dont see viewAllText in the README. is this something we want to add back?

Reply:
sigh yeah we should add it all the deprecated config is kinda upsetting tbh